spec/govuk_design_system_formbuilder/builder/select_spec.rb

```diff
@@ -70,7 +71,13 @@
       specify 'select tag should have the extra attributes' do
         select_tag = parsed_subject.at_css('select')
         extract_args(extra_args, :output).each do |key, val|
-          expect(select_tag[key]).to eql(val)
+          if key == 'data' && val.respond_to?('each')
+            val.each do |dataKey, dataVal|
+              expect(select_tag[%(data-#{key})]).to eql(dataVal)
```

Comment:
The linter has pointed out that `dataKey` is unused and I'm using the wrong variable here - why is this test passing?

Reply:
The linter is complaining because in Ruby we'd typically use snake case (e.g., `data_key`).

This bit of code is actually never reached, the value returned from `extract_args(extra_args, :output)` is `{:required=>"required", :autofocus=>"autofocus", :data=>nil}`, so it's skipping to the `else` below.

I'll pick it up from here.

Reply:
Great thanks!